mesonwrap/githubdb.py

```diff
@@ -99,8 +99,9 @@ def _get_asset(org: Organization,
             with requests.get(asset.browser_download_url) as rv:
                 rv.raise_for_status()
                 return rv.content
-    _log.error('Asset not found project=%s branch=%s revision=%d label=%s',
```

Comment:
I don't think logging commands should be fixed like that. They are defined using `%` operator and this change makes the code harder to read.

```
error(msg, *args, **kwargs) method of logging.Logger instance
    Log 'msg % args' with severity 'ERROR'.

    To pass exception information, use the keyword argument exc_info with
    a true value, e.g.

    logger.error("Houston, we have a %s", "major problem", exc_info=1)
```

Reply:
Ok, I have reverted that change.

Reply:
Ping @legeana .